### ISSUE TITLE
[Feature] Double Tap To Nudge Feature

### ISSUE THREAD

**Issue by karanaditya993** [TelegramMessenger/Telegram-iOS#451]
Hello Telegram team! First, wanted to say that I love this app. It's a great messenger tool and has helped me and my friends stay connected for quite some time. 

This is a feature request that we had a lot of fun with and got utility out of from our previous messenger app, [Hike](https://www.hike.in/)

It's called the [Nudge](https://hike.zendesk.com/hc/en-us/articles/230604247-Nudge?mobile_site=false) feature, where if you double tap anywhere on the chat screen, you nudge your fellow chat mates to bring it to their attention (much like a vibrating notification would do). 

I'd be happy to try and implement this myself via a PR (was looking at the code and saw the [file](https://github.com/TelegramMessenger/Telegram-iOS/blob/7fb0b48ded4b83297453444c9aba89139c4a726a/submodules/Display/Source/TapLongTapOrDoubleTapGestureRecognizer.swift) that I think would be of use, however I am brand new to Swift and I'm afraid it'd take me a while to get something up. 

Thanks again for your work on this!

**Comment by Shreshna1397** [comment]
Yes please. I would very much like this feature to be added. I have become so addicted to using nudge by default, that it feels something is missing!! Please please request you to add this feature. Will be forever grateful!!

**Comment by karanaditya993** [comment]
Wondering if there's any update on this? Happy to implement were I to be pointed in the right direction. Thanks.
